app/Member.php

```diff
@@ -67,6 +73,14 @@ class Member extends Model
     public static $options_allowed_membership_type = array('1 year', 'Life');
     public static $options_membership_type_with_cost = array('1 year (£8)', 'Life (£15)');
 
+    public function setAllVisible()
+    {
+        $this->addVisible($this->getHidden());
+        $this->addVisible($this->guarded);
+        $this->addVisible($this->appends);
```

Comment:
This makes all other attributes not listed in ```$visible``` visible. But i think if I rename the ```$hidden``` array to something else ```$this->getHidden()``` will return all attributes not in ```$hidden``` so I probably can remove those lines.